packages/composites/multi-address-list/src/endpoint/address-list.ts

```diff
@@ -0,0 +1,90 @@
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { config } from '../config'
+import { addressListTransport } from '../transport/address-list'
+import {
+  PoRAddressEndpoint,
+  PoRAddressResponse,
+} from '@chainlink/external-adapter-framework/adapter/por'
+import { walletParameters as anchorageParams } from '@chainlink/anchorage-adapter'
+import { walletParameters as coinbasePrimeParams } from '@chainlink/coinbase-prime-adapter'
+import { walletParameters as bitGoParams } from '@chainlink/bitgo-adapter'
+
+export const inputParameters = new InputParameters(
+  {
+    anchorage: {
+      required: true,
+      type: {
+        ...anchorageParams.definition,
+        endpoint: {
+          type: 'string',
+          description: 'Endpoint name to make a request to',
+          default: 'wallet',
+        },
+      },
+      description: 'Input parameters for the Anchorage EA',
+    },
+    bitgo: {
+      required: true,
+      type: {
+        ...bitGoParams.definition,
+        endpoint: {
+          type: 'string',
+          description: 'Endpoint name to make a request to',
+          default: 'wallet',
+        },
+      },
+      description: 'Input parameters for the Bitgo EA',
+    },
+    coinbasePrime: {
+      required: true,
+      type: {
+        ...coinbasePrimeParams.definition,
+        endpoint: {
+          type: 'string',
+          description: 'Endpoint name to make a request to',
+          default: 'wallet',
+        },
+      },
+      description: 'Input parameters for the Coinbase Prime EA',
+    },
```

Comment:
That's a lot of required inputs, will all 3 always be used?

Reply:
So right now all 3 custodians will be used. The future is unknown, maybe it will be 2 or 4 :)

Reply:
would it work if we set the top level ones to `required: false` and only requested from the ones present? I think this would probably be better for future expansion

Reply:
done